src/backend/partaj/core/api/referral.py

```diff
@@ -402,22 +402,46 @@ def change_urgencylevel(self, request, pk):
                 },
             )
 
+        # Get the referral itself
+        referral = self.get_object()
+
+        referral.change_urgencylevel(
+            new_urgency_level=new_referral_urgency,
+            new_referralurgency_explanation=request.data.get(
+                "urgencylevel_explanation"
+            ),
+            created_by=request.user,
+        )
+        referral.save()
+
+        return Response(data=ReferralSerializer(referral).data)
+
+    @action(
+        detail=True, methods=["post"], permission_classes=[UserIsReferralUnitOrganizer]
```

Comment:
I'm thinking a referral's author should be able to close their own referral. What do you think?